test/cpp/interop/xds_interop_client.cc

```diff
@@ -214,21 +290,47 @@ class LoadBalancerStatsServiceImpl : public LoadBalancerStatsService::Service {
   }
 };
 
-void RunTestLoop(const std::string& server,
-                 std::chrono::duration<double> duration_per_query) {
+void RunTestLoop(std::chrono::duration<double> duration_per_query) {
+  std::vector<std::string> rpc_types = absl::StrSplit(FLAGS_rpc, ',');
+  // Store Metadata like
+  // "EmptyCall:key1:value1,UnaryCall:key1:value1,UnaryCall:key2:value2" into a
+  // map where the key is the RPC type and value is a vector of key:value pairs.
+  // {EmptyCall, [{key1,value1}],
+  //  UnaryCall, [{key1,value1}, {key2,value2}]}
+  std::vector<std::string> rpc_metadata = absl::StrSplit(FLAGS_metadata, ',');
```

Comment:
Suggest using `std::vector<absl::string_view>` here, to avoid unnecessary copies.